### DISCUSSION TITLE
Which file extensions / types does copilot support?

### DISCUSSION THREAD

**Discussion by busti** [community/community#9845] [Copilot Conversations]
I have noticed that copilot does not show any suggestions on specific file types, is there a general list somewhere which lists all the contexts / file-types where it is enabled?

**Comment by thethiny** [comment]
You can enable it on the files you want. For me I have it on for yml/txt files so that it can continue writing for me

**Reply by Petros626** [reply]
how?

**Reply by thethiny** [reply]
> how?

Click on the copilot icon on the current open file on the bottom, next to the type of the file, then enable it.

**Reply by busti** [reply]
This is quite an old thread. At the time it was written there were restrictions on what files copilot would function in.
